pkg/gce-pd-csi-driver/controller.go

```diff
@@ -293,12 +317,12 @@ func (gceCS *GCEControllerServer) ControllerPublishVolume(ctx context.Context, r
 
 	deviceName, err := common.GetDeviceName(volKey)
 	if err != nil {
-		return nil, status.Error(codes.Internal, fmt.Sprintf("error getting device name: %v", err))
+		return nil, status.Errorf(codes.Internal, "error getting device name: %v", err)
 	}
 
 	attached, err := diskIsAttachedAndCompatible(deviceName, instance, volumeCapability, readWrite)
 	if err != nil {
-		return nil, status.Error(codes.AlreadyExists, fmt.Sprintf("Disk %v already published to node %v but incompatbile: %v", volKey.Name, nodeID, err))
+		return nil, status.Errorf(codes.AlreadyExists, "Disk %v already published to node %v but incompatbile: %v", volKey.Name, nodeID, err)
```

Comment:
I'm not gonna make you do this now but in the future this type of unrelated change should go in a seperate PR

Reply:
+1

The added diffs make it harder to review this PR.